azure-pipelines.yml

```
@@ -53,10 +53,10 @@ stages:
           # For public or PR jobs, use the hosted pool.  For internal jobs use the internal pool.
           # Will eventually change this to two BYOC pools.
           ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-            name: Hosted VS2017
+            name: Hosted VS2019
```

Hmm, this syntax is pretty pretty old and I don't think it's supported anymore.

```suggestion
            vmImage: windows-2019
```

This is the reason PR validation is not running. It's trying to be queued but having pool problems: https://dnceng.visualstudio.com/public/_build/results?buildId=1962467&view=results

---

[More details on the vmImage syntax](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#use-a-microsoft-hosted-agent)